sections/clause_0_front_material.adoc

```diff
@@ -1,10 +1,27 @@
 [abstract]
 == Abstract
 
-This best practice document on UML to JSON encoding rules is established as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+This best practice document on UML to JSON encoding rules is initiated as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+
+Application schema in UML are used to model geospatial information as part of data specifications defining information and data content of domain related universe of discourse. In the geospatial domain the ISO 19103:2015 and ISO 19109:2015 UML profile is used, which therefore is also used in this document. Application schema operate at the conceptual level. At the implementation or data level JSON is one of the major data encodings used by current web applications. The UML to JSON encoding rules defined in the context of this document is a best practice. The aim of this document is to support the development of an international standard in this domain.
+
+To facilitate a proper JSON encoding an extension on the ISO 19103 and 19109 UML profile is proposed resulting in a UML-JSON encoding profile consisting of a list of extensions, typically of UML-stereotypes and related tagged values.
+
+The encoding rules are structured in 40 requirement classes and a number of recommendations, subdivided into 18 core requirements, 1 requirement specific for plain JSON schema format, 5 for GeoJSON format, 5 for JSON-FG Schema format, 3 requirements for binding and referencing of elements, 2 for union constructs, 5 for code lists and 1 for encoding a dedicated entity property in JSON. The requirement classes are supported by UML examples and subsequent JSON encodings.
+
+As this is a best practice document it does not include an abstract test suite on conformance classes.
```

Comment:
Omit this. It is irrelevant for an abstract, imho. We state and explain this in the conformance chapter (and Annex A).

Reply:
Suggest to leave it in because it gives the interested and knowledgeable reader a quick overview.

Reply:
If you mean line 12. That can indeed be left out.

Reply:
Yes, line 12. Sorry, when I added the comment - via the blue "+" button - I thought that would automatically only use that line, but apparently it takes the whole block, for some reason.

Reply:
done